automatic/qtcreatorcdbext/qtcreatorcdbext.nuspec

```
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>qtcreatorcdbext</id>
+    <version>4.13.2</version>
+    <packageSourceUrl>https://github.com/admiringworm/chocolatey-packages/tree/master/automatic/qtcreatorcdbext</packageSourceUrl>
+    <owners>AdmiringWorm</owners>
+    <title>Qt Creator CDB Extension</title>
+    <authors>Qt Project</authors>
+    <projectUrl>http://wiki.qt.io/Qt_Creator</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/AdmiringWorm/chocolatey-packages@35eb3e49edb3411d779f64e57ef023abac8a3a06/icons/qtcreator.png</iconUrl>
+    <copyright>© 2020 The Qt Company Ltd.</copyright>
+    <licenseUrl>http://code.qt.io/cgit/qt-creator/qt-creator.git/plain/LICENSE.GPL3-EXCEPT?h=4.1</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>http://code.qt.io/cgit/qt-creator/qt-creator.git/</projectSourceUrl>
+    <docsUrl>https://doc.qt.io/qtcreator/</docsUrl>
+    <mailingListUrl>http://lists.qt-project.org/mailman/listinfo/qt-creator/</mailingListUrl>
+    <bugTrackerUrl>https://bugreports.qt.io/browse/QTCREATORBUG</bugTrackerUrl>
+    <tags>c++ qt ide qtcreator qtcreatorcdbext admin</tags>
+    <summary>CDB (Microsoft Console Debugger) extension for Qt Creator.</summary>
+    <!-- Do not touch the description here in the nuspec file. Description is imported during update from the Readme.md file -->
+    <description><![CDATA[Qt Creator CDB Extension adds support for the Microsoft Console Debugger (CDB) to Qt Creator, a cross-platform, complete integrated development environment (IDE) for application developers to create applications for multiple desktop and mobile device platforms, such as Android and iOS.
+]]></description>
+    <releaseNotes>[Software Changelog](http://code.qt.io/cgit/qt-creator/qt-creator.git/plain/dist/changes-4.13.2.md)
+[Package Changelog](https://github.com/AdmiringWorm/chocolatey-packages/blob/master/automatic/qtcreatorcdbext/Changelog.md)</releaseNotes>
+    <dependencies>
+      <dependency id="qtcreator" version="4.13.2" />
```

Please change this to:

```suggestion
      <dependency id="qtcreator" version="[4.13.2]" />
```

This will ensure that the cdb extension only accepts the exact version of qtcreator (which would be appropriate since they are updated/provided at the same time).

This is of course something that will need to be also be updated in the `update.ps1` script to replace the version during update.

See https://github.com/AdmiringWorm/chocolatey-packages/blob/develop/automatic/privazer/update.ps1#L8-L10 as an example for that.

---

Done, thanks for the link to the example, that was very helpful!